src/Microsoft.VisualStudio.AppDesigner/Common/Utils.vb

```diff
@@ -660,6 +660,10 @@ Namespace Microsoft.VisualStudio.Editors.AppDesCommon
                     userTask.Properties("vs.projectsystem.appdesigner.page-opened.tabtitle") = tabTitle
                 End If
 
+                If alreadyOpened Then
+                    userTask.Properties("vs.projectsystem.appdesigner.page-opened.alreadyopened") = alreadyOpened
+                End If
```

Comment:
Would it be useful to add this property unconditionally? As it stands, the property is omitted when `alreadyOpened` is false, and will only be present with a true value. Maybe allow the value to be both true and false. I guess this comes down to how the telemetry data is processed, so may be a question for @jjmew.

Reply:
If I am reading this correctly we will get the Page-Opened event every time that the user attempts to launch it. This will be even if the properties pages is already opened. I think it fine like it is right now. 

---
In reply to: [563556072](https://github.com/dotnet/project-system/pull/6861#discussion_r563556072) [](ancestors = 563556072)

Reply:
The only place where we pass it as `true` is on the `AppDesignerAlreadyLoaded`, and it's `false` by default. I think Drew's comment makes sense and could better describe the event, letting us know explicitly when they were closed. Doesn't harm to have it 😉